docs/data/material/components/checkboxes/IndeterminateCheckbox.tsx

```diff
@@ -40,6 +41,7 @@ export default function IndeterminateCheckbox() {
             checked={checked[0] && checked[1]}
             indeterminate={checked[0] !== checked[1]}
             onChange={handleChange1}
+            indeterminateIcon={<IndeterminateCheckBox />}
```

Comment:
I think it's fair to say that most users won't ever customize the indeterminate icon and many users copy the demo code to get started. For these reasons, I think it's better to not modify this demo. Otherwise, users might think they need to use the `indeterminateIcon` when using the `indeterminate` prop.

You're already pointing to the `indeterminateIcon` in the paragraph, I think that's enough.

If we really want to have `indeterminateIcon` in a demo, we could have it in the `IconCheckboxes.tsx` demo (under the "Icon" section).

Reply:
Thanks for the input!
I'll update the PR with the suggested changes and remove the indeterminateIcon usage in the demo code.